src/scrapers/otsar-hahayal.js

```diff
@@ -184,18 +185,18 @@ async function waitForPostLogin(page) {
   // TODO check for condition to provide new password
   return Promise.race([
     waitUntilElementFound(page, 'div.lotusFrame', true),
-    waitUntilElementFound(page, 'div.fibi_pwd_error', true),
+    waitUntilElementFound(page, '#validationMsg'),
   ]);
 }
 
 class OtsarHahayalScraper extends BaseScraperWithBrowser {
   getLoginOptions(credentials) {
     return {
-      loginUrl: `${BASE_URL}/LoginServices/login2.do?bankId=OTSARPRTAL`,
+      loginUrl: 'https://online.bankotsar.co.il/MatafLoginService/MatafLoginServlet?bankId=OTSARPRTAL&site=Private&KODSAFA=HE',
```

Comment:
can you use `${BASE_URL}` instead of duplicating the url?

Reply:
done